emcc.py

```diff
@@ -1242,6 +1242,12 @@ def check(input_file):
       if shared.Settings.WASM:
         shared.Settings.BINARYEN = 1 # these are synonyms
 
+        # When only targeting wasm, the .asm.js file is not executable, so is treated as an intermediate build file that can be cleaned up.
+        if shared.Building.is_wasm_only():
+          asm_target = asm_target.replace('.asm.js', '.temp.asm.js')
```

Comment:
Are you saying you want to use the asm.js file from an asm.js build together with js code from a wasm build? That will probably not work, as if we know in the second build that there will be no asm.js, we depend on that. That's why we have the `BINARYEN_METHOD=asmjs,wasm-native` etc. if you do want a fallback.